### PR TITLE
upgrade to 7.62.0.Final

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: data/pom.yml [hunk 1]
```diff
@@ -11,70 +11,70 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
...
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-integration-distribution-7.62.0.Final.zip
 
-    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-eap7.war
-    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-wildfly23.war
-    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.61.0.Final/drools-wb-jcr2vfs-distribution-7.61.0.Final.zip
-    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.61.0.Final/kie-wb-example-repositories-7.61.0.Final.zip
+    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-eap7.war
+    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-wildfly23.war
+    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.62.0.Final/drools-wb-jcr2vfs-distribution-7.62.0.Final.zip
+    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.62.0.Final/kie-wb-example-repositories-7.62.0.Final.zip
```
Comment (mareknovotny) [inline data/pom.yml:65]: i can't find this URL, is it still in released assemblies? I can't see it even in previous releases

File: data/pom.yml [hunk 2]
```diff
@@ -11,70 +11,70 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
...
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.61.0.Final/droolsjbpm-integration-distribution-7.61.0.Final.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-integration-distribution-7.62.0.Final.zip
 
-    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-eap7.war
-    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-wildfly23.war
-    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.61.0.Final/drools-wb-jcr2vfs-distribution-7.61.0.Final.zip
-    droolsWorkbenchExampleGitReposZip: https://download.jboss.org/drools/release/7.61.0.Final/kie-wb-example-repositories-7.61.0.Final.zip
+    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-eap7.war
+    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-wildfly23.war
+    droolsWorkbenchJcr2vfsZip: https://download.jboss.org/drools/release/7.62.0.Final/drools-wb-jcr2vfs-distribution-7.62.0.Final.zip
```
Comment (mareknovotny) [inline data/pom.yml:64]: i can't find this URL, is it still in released assemblies? I can't see it even in previous releases

File: data/pom.yml (hunk 1 above)
Comment (mbiarnes) [reply data/pom.yml:65]: checking

Comment (mbiarnes) [reply data/pom.yml]: @mareknovotny a business-central-{$kie-version}-eap7.war is not any more provided - this for long time! But is is still in the the pom. You have not displayed this link in the drools webpage (www.drools.org).
Also in the latest - there still were not working links but also they didn't get displayed in the web. Have removed these.
Another thing is iof we still need the `latest` - sinc the links in the web are pointing to `latest.final`

File: data/pom.yml (hunk 2 above)
Comment (mbiarnes) [reply data/pom.yml]: @mareknovotny a business-central-{$kie-version}-eap7.war is not any more provided - this for long time! But is is still in the the pom. You have not displayed this link in the drools webpage (www.drools.org).
Also in the latest - there still were not working links but also they didn't get displayed in the web. Have removed these.
Another thing is iof we still need the latest - sinc the links in the web are pointing to latest.final

File: data/pom.yml [hunk 3]
```diff
@@ -11,70 +11,59 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
...
     # droolsjbpm-tools iss not released any more, last stable version is 7. 46.0.Final
     # latestDroolsJbpmToolsZip: https://download.jboss.org/drools/release/7. 46.0.Final/droolsjbpm-tools-distribution-7. 46.0.Final.zip
 
-    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.61.0.Final/droolsjbpm-tools-distribution-7.61.0.Final.zip
-
-    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.61.0.Final/drools-docs/html_single/index.html
+    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-tools-distribution-7.62.0.Final.zip
 
-    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.61.0.Final/kie-api-javadoc/index.html
+    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.62.0.Final/drools-docs/html_single/index.html
```
Comment (tarilabs) [inline data/pom.yml:30]: Can you kindly explain these YAML `key` changes?

Comment (mbiarnes) [reply data/pom.yml:30]: @tarilabs if you look at line 32 you will find KIE_API_documentationJavadoc pointing to 7.62.0.Final

Comment (tarilabs) [reply data/pom.yml:30]: oh i See now, thanks! 👍

**Review by tarilabs — APPROVED** [review]

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**